jobs/wireless/designs/0001_design.yaml.j2

```diff
@@ -0,0 +1,612 @@
+---
+location_types:
+  - "!create_or_update:name": "External"
+    content_types:
+      - "!get:app_label": "dcim"
+        "!get:model": "controller"
+  - "!create_or_update:name": "Region"
+    nestable: true
+    children:
+      - "!create_or_update:name": "Site"
+        content_types:
```

Comment:
Does design builder support adding to m2m fields? It would be nice if we could just add dcim.device and dcim.controller here and not have to be concerned with the content types that other designs require

Reply:
It looks like it, I removed the extra content-types